### ISSUE TITLE
Unnecessary dialog about finishing the pyramid

### ISSUE THREAD

**Issue by christopho** [wrightmat/zbom#17]
When you leave the desert without having found all pieces in the pyramid, a dialog says that you should go back. This can be helpful, but:
- The dialog also pops when entering the desert, coming from the kakariko village.
- It feels weird that the game gives a hint like this. It should only be the gerudos who tell that to the player.

Additionally, now that there is a hint about the eyes in the room where the player has to shoot arrows, the risk that a player wrongly thinks he cannot finish this dungeon yet is not huge.
And even if is happens, by talking to various NPCs, everything indicates that the next step is still the pyramid.

So I suggest to simply remove this dialog :)

**Comment by wrightmat** [comment]
I've already corrected the error where the dialog pops up when coming from Kakariko, it now only fires when leaving the Pyramid. The idea I was trying to convey was that Ordona is the one reminding the player, so it would be a little less weird. I feared that the player might miss one part, leave the desert and have to go all the way back.

I'm amenable to removing the dialog, I just wanted to provide what I was thinking. Thoughts now?
